tests/pytest_tests/system_tests/test_sweep/test_wandb_sweep.py

```diff
@@ -169,3 +169,27 @@ def test_minmax_validation():
 
     with pytest.raises(ValueError):
         api.api._validate_config_and_fill_distribution(sweep_config)
+
+
+def test_add_run_to_existing_sweep(user, wandb_init, relay_server):
+    # Test that updating settings with sweep_id includes it in upsertBucket mutation
+
+    with relay_server() as relay:
+        sweep_id = wandb.sweep(SWEEP_CONFIG_GRID, entity=user)
+        settings = wandb.Settings()
+        settings.update({"sweep_id": sweep_id})
+        run = wandb_init(entity=user, settings=settings)
+        run.log({"x": 1})
+        run.finish()
+
+    for query in relay.context.raw_data:
```

Comment:
could you add it as utility? you could see how we do the other fields, also you probably want to make sure that you are using the correct `run_id` when you do it as a utility. 
I think it might be useful for others and if we change the query it would be nice to change it in one place. 

refer to this https://github.com/wandb/wandb/blob/main/wandb/testing/relay.py